client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/ILMDocumentationIT.java

```diff
@@ -797,32 +797,11 @@ public void testAddSnapshotLifecyclePolicy() throws Exception {
         // end::slm-put-snapshot-lifecycle-policy-response
         assertTrue(putAcknowledged);
 
-        // tag::slm-put-snapshot-lifecycle-policy-execute-listener
-        ActionListener<AcknowledgedResponse> putListener =
-                new ActionListener<>() {
-            @Override
-            public void onResponse(AcknowledgedResponse resp) {
-                boolean acknowledged = resp.isAcknowledged(); // <1>
-            }
-
-            @Override
-            public void onFailure(Exception e) {
-                // <2>
-            }
-        };
-        // end::slm-put-snapshot-lifecycle-policy-execute-listener
-
-        // tag::slm-put-snapshot-lifecycle-policy-execute-async
-        client.indexLifecycle().putSnapshotLifecyclePolicyAsync(request,
```

Comment:
I think rather than remove this (which we need for documentation), we should relax or remove the assertion to not trip on the policy version.